### PR TITLE
[WIP] Lookup Account & Controllers for Sagas

### PR DESCRIPTION
**Pull request by TimWDev**

### CODE REVIEW

**Comment by TimWDev** [comment]
Everything is in place now I think, still on the block:
- Candid interface updates
- Unit tests to cover new logic
- Update deploy script if needed

File: actors/account_registry/src/main.rs [hunk 1]
```diff
@@ -13,22 +17,100 @@ thread_local! {
...
+    ACCOUNT_REGISTRY.with(|reg| reg.borrow().get_accounts_for_controller(controller))
+}
+
+#[query]
+pub fn lookup_account_controllers(account_id: String) -> Result<Vec<String>, String> {
+    ACCOUNT_REGISTRY.with(|reg| reg.borrow().get_controllers_for_account(account_id))
+}
+
+#[query]
+pub fn process_request(request: Envelope<String>) -> () {
```
Comment (SuddenlyHazel) [inline actors/account_registry/src/main.rs:69]: This needs to be an `#[update]`

Comment (SuddenlyHazel) [reply actors/account_registry/src/main.rs:69]: Else we can't send intercanister calls.

**Review by SuddenlyHazel — COMMENTED** [review]
This looks great so far! Let's add the work to respond to the controller queries in the channel handlers now!

File: actors/account_registry/src/main.rs (hunk 1 above)
Comment (TimWDev) [reply actors/account_registry/src/main.rs:69]: Good catch!

File: src/rs_quark/event_stream/stable/broker.rs [hunk 2]
```diff
@@ -91,12 +93,14 @@ impl StableBroker {
         return out;
     }
 
-    pub fn mark_task_complete(&mut self, id: u64) -> () {
-        self.complete_task(id)
+    pub fn mark_task_complete(&mut self, id: u64, clock: ClockFn) -> () {
+        self.complete_task(id);
+        self.enqueue_next_round_of_notify_tasks(clock);
```
Comment (SuddenlyHazel) [inline src/rs_quark/event_stream/stable/broker.rs:98]: Bug fix @TimWDev

**Comment by TimWDev** [comment]
Event broker unit tests failing after fix, other than that LGTM

**Pull request merged**